pandas/tests/io/json/test_normalize.py

```diff
@@ -472,9 +472,9 @@ def test_usecols(self, nested_input_data):
         expected_df = DataFrame(data=expected)
         tm.assert_equal(result, expected_df)
 
-    def test_usecols_with_list(self, nested_input_data):
+    def test_use_keys_with_list(self, nested_input_data):
```

Comment:
Can you parametrize the list / str / callable use cases instead of creating separate tests?